src/reconstruction/modelGeneration/massBalance/eleMatrixToFormulae.m

```diff
@@ -0,0 +1,66 @@
+function metForm = convertMatrixFormulas(element, metEle, dMax)
+%Convert the matrix form of the chemical formulae into string form
```

Comment:
Could you update the documentation to fit the documentation template (in order for the autodoc being able to process it)